src/main/java/org/coffeetrain/doggifs/DogGifRepository.kt

```diff
@@ -0,0 +1,31 @@
+package org.coffeetrain.doggifs
+
+import android.content.Context
+import java.io.InputStream
+import java.util.Arrays
+
+class DogGifRepository(val context: Context) {
+  private val assetGifs = Arrays.asList("pounce", "weiners")
+  private var currentIndex = 0
+
+  fun getGif(handle: String): ByteArray {
```

Comment:
if you named this `get` and applied `operator`, you'd get `repo[handle]` syntax.